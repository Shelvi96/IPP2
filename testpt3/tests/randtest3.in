NEW ID

663209761>7
28110>6673712
0;55>4;06425:75
;854154734>2559
4>3
6>:4:
:0;79>3
78;3827;8>45310:7645
7:1798>04;;4;205:
278;>9
88>451;36;56
927076270>;6;;510
18761099:0>883
43:25962>05;:;
70>6:
:8:8>049771:
73>293233
22362>53;517
5460;;>5750973
::>45;271;
08306>4
0>0:47
44>0;42707;3
0244;7453>9
895041>031483;0
2352::;1>3:
87>452817962
:261394>6:889
657;13>9019587331
5;78>;77933;11
0>978
;>7526
586:71;84>3
1:420070>207263
89;5>575:
12448>64069;;
2;6>91752
204:56869>29
8133236159>170042:766
;119:6;;>841378
8567:57::1>587:78356
8>;810;88045
;34:1202;8>175543
10691;>2;
654:01>270870580
5894993;2>4459326
37592651>2::87
5086>;897
2989>1026
846:7005>741
57287;40>1038572
8:868>6852893362
719>2346:409
858>3
495931::95>74647
0;6:7:856>131:7:48
964639573>23259
00096:;>;901
873301>176498;
20760847:7>9
1::9:50>:;3
5;61>172
23>:275:413
314856749>1037:4211
82:>97019
80541:;>7048177
4289:4>12983
5;3:>7;66;
7320>933;00
847>23651583
32>211147
7882>0729;
8146:5>6842238781
36;4220>16
0702;:4>1;9950
5;1>3407869
92:>2558
31>843597627
8:305466>8871;4;
120734;816>755
4971>6852
0>;96;
1;:>4
49>;91449
4::::4>;8;
9>10;
:39;7:::>3300
4>;657:479
5745>971:69947
8280343>:91
5>:8555
06915>2642204;47
8169>623430:07
:873429>44003
0245850>07
3;1767>33
;9>;1235
;9>43013;12;
14;80>37:64859
864748>7152068339
6>430;1:
23>9
:5>62:1777723
95>32
;7:4>561178
1;:7232939>64037866;1
3>12;
1>:2;8953;3
1:55;85>6;2;49:9
1011:75:>:185
945>35
6>3
;;99913>64
19>:0675689
349729;2>:4:75
8;9;4;>7;;
49261:::9>;87;
;1:46;74>477;:
16770>53:3413
1636>;1
79:>8926770721
08:3>;
345:;>:4:
50>1:2
99;4>40302:01
8:6:9885:9>29;4;:209
0>272467:
78159962>34
17:454>66
09435:2351>6:0742:0
455>9
:694395>4;2:68;66
:661463>;
61;87053>:3619
89864:>:596;1;
294163>;689163
7;845421>;;530
62127755:>105
3>7
93;1:>893
039:>:09:
2828070>8;95
06>16
8;708>1;6294
9:5:5555>3:;79
0>79721372
5>3125
7394>20350
64>1;0851
58:25>47;76
4>6;2
8252783:9>8:0
6:738;:7>853:02
2277>6385445
46>61
35;2>88;:82;
4;1154121>789513;::8
1069;49>8;
@4340471129:0;5;947;9;96:2782::125521442;14;41269956034:9
@:3908;556:::413000226333:7:25;28238272814:;9;69;
@35470897:;4:5041978339710846:355090491;857:28239134
@0314053208709047177:791734458050379700150959128617584678235:3:3;506:4874945::2409906786:;;071
@6;709;083109;;383493:5383773
@9;88609:551692:856;2770029718339235086:534;1699471630:76;388:354;34797453361575003702375:718:;4
@608770028641695651;;8:4:5::712972522:74658;;:4:39929;:7885317082:348:063957;152;6805:85;180199
@;4;72;190:4;;9527534439386397446835:7:87847;6461:9624390102896652;899849039683064608
@90312;:9:4;940;0404893843201202;15079347;7837774034160
@92::8;5;4:3;76865330327;2;34972;101936875;7463;3:64
@83436;;461375481;48887
@6239195::975364;59242;13294:5;2;2:17;69174633:287902055231481:070160;31;882;68;6938195347;4024;29
@21080046326204:85;6135475648955;6576:3416;8636709150::
@74731185;54:78296154833948246071974:047;2099806653:2
@6;225:9:::75399;5815:;529375193;0:260;8:9374084955278;9624;82672890988;:;;247610487
@;86219238994;:5;659:402;:742733;0111378;457731726402865759955
@4565:90688;3478396718472;03058191;6;8:98;133806965:795909417168653655
@1484;88957332403927:90857505
@603:9;258;372:9;;4853:24:;;3085:8809;23;2:68877;;74796;7622
@:78:78873365155
@051405;24292840:;08;04:3;4009622;3630554;767;6
@::11;:;29832994;0;53;214182079597;;:1:12:84890
@9;543;5;511982362690;;11:1:6::5;2245698;398343161:64:0601464633867509134337;;154;3913359;;15
@4614;51509874765
@3970237846;93659;998220;288161992895;457244:3:7678322318;99:3
@7575261;248:2;8158:0:44;1712541
@18:86370;3;16;74794510920259:62364;087172419381;598::912;:;18652:4263726;37730
@09::;831;15
@9;:;9668:5:895740720802006547299170316;301;9::5:5726;4
@4:30:69;;8;29655:68747064;82210:34:52741748921200:;893326;4849;859579:0868582702:;:7251896813;1
@977:9763803:;3492
@980:89442;44;53:7065769:4910:767366;727:63698987961842603;02:6115;40632078175136;82320363301216752;
@6235:408:729791:944104629112305928255713274:
@5
@628;60581698:11;4:61712:3125609065;09
@8;3975:846::;9;577724162507845408;13433126;133732
@96;704;04390354808371555;98:4245030
@;;5722797;5;0861606516398729222157879788;5;;554;93829;;::5;4;15485395;9488725;172:941164:5
@:6626;:74148:430175817:99335181;6760;4;39747;;;165100;9121:81
@78;10:906
@7:6;16483507999327;4;;4;089283185;;793700781895;88787;;07028878167842786248;51221929059;
@3;1:7386348;1359;874:95;363833
@:2
@82:985598:068323;5488;307:309979;9683453
@99:445791350:9;100:2;;19:142:747973142:6:7:852196;;531216:35:747688;2:905;921207231947:;618400370;:
@97366490717148:988061021050142819;;848839742823427:3708791013865055421730398900;;2666263
@:4;;24;897226529736739;639457816449788393939296104941024
@615663;:06609736:008:2863477:349413382108009737:70:96:3132855422:6560:71:;2:3:822600875;:1
@79;175:6695;;419735253
@90923;70750049;:7076;089;3611:7:251940:39242374;709649331:9204469936:5177913466363:
@555:720;78586968683655923351:7331810350:69:
@658513;:88033645179642770;9687:70:051;42;8562:;8506131074151178158;:;7023:8:076984;3:3229
@3:633;3292924104
@00990107;89258936169416;:073402843:45:38:42300::557965086802
@6;0:983;33:765;:493140;917717:92914:1;942;385269;127614;84082923261752;7228;8
@870:263:5;25;86130;874;645044824726189;506:0685:85639915259:54206817950:3;:9;3;886;:713
@650
@97473:25773::5;98;5999;8:6:83;046901766611436713::4;778916:59:97;196870198538
@:6466012130;0::951;60;600719:93170;145350450671;84:83418;65171621:493;237;71:85;1778304
@62753688805;3;6;310119024037071;9140;007456894705;1692152416365572966:1;3;;0360096;:80428983368
@86964;5;244628:429221:49;553;16;779;:331;7;63926;880752::7118;4471758::09961;0;:8;:74466;;70334:
@3756:5373863;8
@;7;7;92391305;2261;0;58;8894868;5;6898;:2
@2;545094499
@0:19293288569278493688:1005597915::;;25;2:6;11052702323820537244573818077
@703;956971429:2662126:9:6::::5855;878247309
@34;96034:5297575:828;:380;384557948793;;8604;8918097:473833088;94;85275;59745465871;40603943530120
@8847512:;;84739:7:4;5139
@610027923874;5325019908;4856;24:74293;;:0;338:5226;4673;70976192:;;533432;629;4
@97847;72441:9:07:411;9858279501279:29555163;98:7030;084023:73
@13945:12
@31:2:6096419815939724:777;591;4403:29238:49;53804760528515664;2560;37339718588943441:46;
@4:63031529050155225:;167675259
@17:5;16437818::540;23003:;05899:877805071189;663:996::90995:;6875375801911:405;22884:5907
@:616569549597::52221551190;:465246014168:39
@1833:5872:942826670:50;:16:;109384::176497833299:1;71:9680920:5823:4280338
@::7;08:8115
@15939705288::8:20:02082544165:5;822527383361;873:86:883048::78972344:745
@272:658
@3;23278
@96:5
@98214511:339568322619:886;941332353;2405
@8252:8453:65:6;1372::817;5:85
@819570209927601954473238165363:6400006417:8129:75260
@119;616930634:3:2854466145;:;;60083719442:86839202:414:9918006412803580;;86;3323940:928:3
@;7:305;0099088867411;;631759905888485715426526;1:17:45150;7934:3064515;:;9313662;
@43691:880442758;2985772206970804471992916131:;51916891;9788;89348815227042622;7;06;17:
@6:;5780348923;13339:98501;02::149598193::0010547069:27:3::691;12432:058:603:509967706684
@61997;6:1
@2;459487108:676;147:4132::580;03:40;0061160;2::7655::941;2901:736727700864802
@804
@;66776996:;555074;;58:98:4::4687123080
@638;06;8:;;7;99;019:94:9910994;4;7305704:;3110415537124:38044;0468807885;3
@082
@1718930;:8;65:7993
@1:51784510;66:72727;07:7016129:3;7864435523303
@054;:41986
@:848374;104451:04:35:1:1;67021702315:593557265227:7:;9;2366:81:28376889518;;
@61041:7:216874806:8::6:638037;85109638859254:68988762959550580:95078845:62698365;53660
@498160;7933947225982530279:14:800816141:88803261;27968;599650650
@6683;:;8:331944301101612472342:;0:20050;874948082283095580012:430:703;33670:846::65382042099
@142;;66229011;9:9;884;47;01;2:53273561;8;094;;20:;0
@28261793636021:839869::0115;4566204382:654:;:00128;;:94;21366150297235082:803855844;102415207741
@030931
@513818110:062324
@447408543926:50310412
@64:4656262:5:;2735594778;8871257:8;51977061
@576043:8:5811512;62:6178;67340517324604
@51:::34926;80:8394618;34390114::5103445634377;:47854;00610828:02;0989224;9;611:0174008
@10;499831909:3322510662:7177
@3:389;
@77008496740191577766916:84
@428118514:23567;498;84575617755;;1509:568716289:959:175616603;03057271088122130;82
@:968:42;753260207303359:60:5106;14;08638;:292491;19766
@1;7:00531125:00:02094976883371080860037152;72;5269
@:6;43;;:61;61;1638181370:;505
@431708;;;90:4619049143903731;0843937:2;9
@;;0549906;41085;;6:;66772654520950299
