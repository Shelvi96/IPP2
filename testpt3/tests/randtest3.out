0
4584664420663164928
8646911284551352320
0
23830830080529032
0
0
0
0
8646911284551352320
622871552
0
0
11529215046068469760
0
0
0
22767782221183549
15276573281352017886
127
4515421576392343552
16033974980538075285
0
3420
0
3875232013000000000
0
0
0
0
57611
0
2152506769996977501
0
18339792599843340288
18122484900538875904
6276274362068762624
0
0
0
0
565214511000000000
0
16752270939773507882
0
0
0
0
5762242001
0
9347112333191262526
0
3539
0
0
0
0
0
0
0
0
24
14310270829097648128
0
0
13090407208301101056
0
537434311000
0
0
0
0
2929928062628926007
0
13090407208301101056
0
0
0
0
0
0
10167566679070536614
47641542120000000
16536945392058083133
0
0
0
0
0
0
0
17163326387315015680
0
0
467324
4515421576392343552
0
0
0
0
0
8646911284551352320
0
0
1912
1009227411
13090407208301101056
3209148629178928681
0
89834838096715776
0
9456207656643
0
0
14555633995661443072
47621532100000000
18248461145050984978
12878782705576212282
